src/jaspObject.cpp

```diff
@@ -490,32 +517,17 @@ bool jaspObject::checkDependencies(Json::Value currentOptions)
 				return false;
 
 		for(auto & keyval : _optionMustContain)
-		{
-			bool foundIt = false;
-
-			for(auto & contains : currentOptions.get(keyval.first, Json::arrayValue))
-				if(contains == keyval.second)
-					foundIt = true;
-
-			if(!foundIt)
+			if (!isJsonSubArray(keyval.second, currentOptions.get(keyval.first, Json::arrayValue)))
```

Comment:
@JorisGoosen I don't think the old code ever worked properly here, so I took the liberty of fixing it. Also, see https://github.com/jasp-stats/jaspTestModule/pull/41 for testing. When I tried the old comparison it didn't work unless I ordered things such that `contains == keyval.second` (but it should also work when `keyval.second ∈ contains`).

Reply:
Aah the `keyval.second ∈ contains` was something I never intended so that didnt work. But I can see how it would be useful yeah.